### ISSUE TITLE
Accessing _info metadata from service ticket

### ISSUE THREAD

**Issue by bartbunting** [sowderca/connectwise-ruby-sdk#3]
Hi,

I am trying to access the values in the _info metadata of a service ticket.

This appears not to be possible currentlyunless I'm missing something.

The values show up in the debug output when the ticket is retrieved:

```
"_info":{"lastUpdated":"2018-03-04T12:22:10Z","updatedBy":"bart","
```

When I try and access via the _info metadata object they do not exist.

```
error: undefined method `last_updated' for #<ConnectWise::Metadata:0x00007f94bddfac10>
```

The code is:
```
puts "Last updated: #{t._info.last_updated}"
```

Looking at the metadata.rb model there appear to be no accessors defined for any of the values that show up in the debug.

Is it a case of simply defining these?

I'm sorry I don't understand how the classes are generated.

Happy to make a patch if it is as simple as defining the missing data.

Any advice welcome.

**Comment by sowderca** [comment]
@bartbunting, yes this appears to be an issue with the schema generation. I believe I have a fix for it, which I'll test and push an update for. Sorry about the wait.

**Comment by sowderca** [comment]
@bartbunting are you still having this problem?
